### ISSUE TITLE
minor bug: keywords are highlighted in comments

### ISSUE THREAD

**Issue by serene-fermi** [dangh/zed-tmux#2]
Keywords like `display-panes` are highlighted in comments. `show` is highlighted too, not sure why tho.

![image](https://github.com/user-attachments/assets/2ff78cf9-d2fd-4720-9123-cb7d228bd5f9)

**Comment by dangh** [comment]
I think the parser failed somewhere before this comment, so it's not treating the comment as is. Could you share your tmux config?

**Comment by serene-fermi** [comment]
This is a minimal working example:

```
bind-key -T copy-mode-vi C-f command-prompt -T search -p "find:" { send-keys -X search-forward "%%" }

set-option -g history-limit 20000

# Set the time in milliseconds for which the indicators shown by the display-panes command appear
set -g display-panes-time 4000
```
[Screencast from 2024-08-30 15-59-32.webm](https://github.com/user-attachments/assets/8901f4d8-c65c-414c-b8fe-286ca0cd7895)

**Comment by serene-fermi** [comment]
I see now why this happens. This is what the extension looks like in Zed for me:

![image](https://github.com/user-attachments/assets/496d134c-5917-4d77-9b1e-7d414a8cd2aa)

See version `0.0.1`. But comments support was added with `0.0.2`: https://github.com/dangh/zed-tmux/pull/1/files. It seems Zed has no access to `0.0.2`, there is no update available.

**Comment by dangh** [comment]
I just made a pull request to update tmux extension to v0.0.2 here https://github.com/zed-industries/extensions/pull/1337

**Comment by serene-fermi** [comment]
I am using now 0.0.2 but the problem does not seem to be fixed.

**Comment by dangh** [comment]
Yeah that's what I think 😛 It has to be fixed upstream. I'm neither an expert in Tree-sitter nor tmux, so I can't fix it, unfortunately. Could you raise an issue here? https://github.com/Freed-Wu/tree-sitter-tmux

**Comment by serene-fermi** [comment]
Sounds like this has been fixed now: https://github.com/Freed-Wu/tree-sitter-tmux/issues/23
